test/bdd/steps/api/search-assertion.js

```diff
@@ -0,0 +1,68 @@
+const { When, Then, Given } = require('@cucumber/cucumber');
+const { expect, assert } = require('chai');
+const utilities = require('../../../utilities/utilities');
+let handlerId, searchResult, searchData, parsedKeywords, lastSearchData;
```

Comment:
Remove local variables, always use this.state to store the data between the steps